test/unit/core/factories/GridOptions.spec.js

```diff
@@ -143,7 +147,10 @@ describe('GridOptions factory', function () {
         rowTemplate: 'testRow',
         gridMenuTemplate: 'testGridMenu',
         extraOption: 'testExtraOption',
-        appScopeProvider : 'anotherRef'
+        appScopeProvider : 'anotherRef',
+        columnHeaderKeyDownAction: 0,
```

Comment:
Rather than using 0, 1 and 2, wouldn't it be easier to read this if you used the constants?